front-end/src/store/analyzeTranscriptSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const analyzeTranscriptSlice = createSlice({
+  name: 'analyzeTranscript',
+  initialState: {
+    analyzedTranscripts: {},
+    override : false,
+    projectIdToBeAnalyzed : ""
+  },
+
+  reducers: {
+    addAnalyzedTranscript: (state, action) => {
+      state.analyzedTranscripts[action.payload.projectId] = action.payload.transcript;
+    },
+    deleteAnalyzedTranscript: (state, action) => {
+      delete state.analyzedTranscripts[action.payload];
```

Comment:
What does this do?

Reply:
Again this is not used right now but it's helpful working with multiple project ids and transcripts. In this particular one I was thinking about the privacy. Like if user checked some projects, wants to move on to the others but might not want the previous project to be available or something like that.